src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs

```diff
@@ -375,6 +375,23 @@ public void ClearRenderEffect(bool disposeEffect = false)
             }
         }
 
+        /// <summary>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ImageUrl GetImageUrl()
+        {
+            IntPtr cPtr = Interop.View.GetOffScreenRenderingOutput(SwigCPtr);
```

Comment:
```suggestion
            IntPtr cPtr = Interop.View.GetOffScreenRenderingOutput(SwigCPtr);
            if (cPtr == IntPtr.Zero)
            {
                return null;
            }
```

I left some comment about invalid texture case. For now, `ImageUrl` dont have any way that given texrure is valid or not. So how about to return nullptr for invalid texture case?